Sources/DeckUI/DSL/Slide.swift

```diff
@@ -15,10 +15,10 @@ public struct Slide: Identifiable {
     let padding: CGFloat
     let comment: String?
     let theme: Theme?
-    let direction: SlideDirection?
+    let direction: SlideTransition?
```

Comment:
We should change the name of the property from `direction` to `transition`, too

Reply:
Oops! Good catch 😊